### DISCUSSION TITLE
Can the MA players be made available as media players to Home Assistant?

### DISCUSSION THREAD

**Discussion by rarroyo6** [music-assistant/support#3245] [Q&A] [answered]
I have several wall tablets with the SnapCast client installed, and I have MA with the SnapCast provider. Everything works fine, the tablets show up as players in MA, and I can control and play music to them.
I read in multiple places that MA can make these players available as media players to Home Assistant, so they can be used for TTS, but I have not been able to find how to do this. I have tried going through the MA settings and googling it, but I have not been able to find anything.
Is it possible to expose the MA players to Home Assistant? And if so, how can it be done?

HA version: 2024.11.4
MA version: 2.3.3

**Comment by OzGav** [comment, marked as answer]
You need the integration https://music-assistant.io/integration/

**Reply by rarroyo6** [reply]
Thank you, I didn't realize there was a separate integration on the HA side.
I just saw that it will be in 2024.12, so I'll wait a couple of days and try it out.
